packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -1506,17 +1499,16 @@ export function downloadDs(node: ZoweDatasetNode) {
             lbl = node.getParent().getLabel().toString() + "(" + node.getLabel().toString() + ")";
             break;
         default:
-            vscode.window.showErrorMessage(
-                localize("download.invalidNode", "downloadDataset() called with invalid node.")
-            );
+            vscode.window.showErrorMessage(localize("download.invalidNode", "downloadDataset() called with invalid node."));
             throw Error(localize("download.invalidNode", "downloadDataset() called with invalid node. "));
     }
     const filePath = getDocumentFilePath(lbl, node);
-    return ZoweExplorerApiRegister.getMvsApi(profile).getContents(lbl, {
+    let a = await ZoweExplorerApiRegister.getMvsApi(profile).getContents(lbl, {
```

Comment:
Instead of defining it and then returning this we can probably just `return ZoweExplorerApiRegister.getMvsApi...`

Reply:
Thanks @traeok that was for debugging purposes my miss.